README.md

```diff
@@ -4,24 +4,65 @@ This is an online multiplayer take on the classic party game Mafia. The game spl
 However, since this is an OSS project, developers are free to edit rules and add new functionality as they please.
 
 <p align="center">
-<img src="https://cdn2.iconfinder.com/data/icons/rounded-white-basic-ui-set-3/139/Photo_Add-RoundedWhite-512.png" height=200; width=200>
+  <img src="https://raw.githubusercontent.com/Mafia-Hands/Mafia/main/frontend/public/logo192.png" height=200; width=200>
 </p>
 
 
 Please review the [documentation](https://github.com/Mafia-Hands/Mafia/wiki) for guidelines and information about the software.
 
 ## Installation
+Before you install and get started with this project, please ensure that you have installed [nodejs](https://nodejs.org/en/) and [npm](https://www.npmjs.com/) on your computer.
 
-Use the package manager [npm](https://www.npmjs.com/get-npm) to install all dependencies.
+This project uses the package manager [npm](https://www.npmjs.com/get-npm) to install all dependencies.
 
-```bash
+First, clone the repository and open the project directory:
+```
+git clone git@github.com:Mafia-Hands/Mafia.git
+cd Mafia
+```
+
+To run this application locally, the frontend and backend must run together. You can do it like so:
+
+**Frontend:**
+```
+cd frontend
 npm install
+npm start
 ```
-## Contributing
+**Backend:**
+```
+cd backend
+npm install
+npm start
+```
+**Note:** You do not need to run `npm install` everytime. Only when a new package is added.
```

Comment:
Could we mention how to run the tests on our local machine? And possibly how to run puppeteer as well :)